### DISCUSSION TITLE
[@types/node] diagnostics_channel name should support symbol or string

### DISCUSSION THREAD

**Discussion by destumme** [DefinitelyTyped/DefinitelyTyped#63098] [Issues with a @types package] [answered]
Currently the types for a diagnostic_channel only allow string names, the name per node's docs can be a string or a symbol. 

https://nodejs.org/docs/latest-v18.x/api/diagnostics_channel.html#diagnostics_channelchannelname

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
see #63116
